cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2804,6 +2804,21 @@
     </description>
   </property>
 
+  <property>
+    <name>i.log.appender.provider.enabled</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>i.log.appender.provider.extensions.dir</name>
+    <value>/opt/cdap/master/ext/i-log-appender</value>
+  </property>
+
+  <property>
+    <name>i.log.appender.provider.stackdriver.project</name>
```

Comment:
Is the plan to have stackdriver log appended as an extension? If so, then having a extension-speicific configuration in the cdap-default.xml is not desirable

Reply:
Removed